README.rst

```diff
@@ -0,0 +1,33 @@
+pymesync
+========
+
+Python module for TimeSync
```

Comment:
We should also add a section in here that you can install via setup.py or pip. This change might happen once it's on pypi.